source/2015-07-14-3-planning-pokers-in-3-days.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+title: 3 planning pokers in 3 days
+date: 2015-07-14 10:41 UTC
+tags:
+---
+
+During our Kalamata Klassenlager there were quite some pitches to try out
+new technologies. As the week progressed three groups around three different
+main technologies seemed to emerge: React / Hoodie, Volt and Meteor.
+Instead of implementing a todo app for the millionth time each group
+implemented a planning poker in their respective technology.
+An online planning poker could actually be something useful.
+
+## React / Hoodie
+**Sourcecode**: [github.com/panter/hoodie-poker](https://github.com/panter/hoodie-poker)  
+**Technologies**: [Hoodie](http://hood.ie), [React](https://facebook.github.io/react)  
+**Online demo**: -
+
+### +
+TODO
+
+### -
+TODO
+
+### Conclusion
+TODO
+
+## Volt
+**Sourecode**: [github.com/panter/volt-poker](https://github.com/panter/volt-poker)  
+**Technologies**: [Volt](http://www.voltframework.com)  
+**Online demo**: [volt-poker.herokuapp.com](https://volt-poker.herokuapp.com)
+
+### +
+The synchronization between all clients works out of the box.
+Being able to write (almost everythin) in ruby. Opal lacks a lot of ruby's standard library though.
+Automatic attribute binding between controller and view, which also means automatic updates on all the clients when something changes.
+
+### -
+Error messages are mostly not very helpful, unless there are no error messages at all.
+No debugging mechanism yet.
+
+### Conclusion
+Volt is still in a very early stage and therefor not everything works yet and debugging is hard. Being able to use ruby as the main language though is very nice. Its concept of abstracting away the messaging layer HTTP facilitates developing an application, as much of the duplication between client and server are not necessary anymore and the developer doesn't need to handle API requests between client and server.
+
+## Meteor
```

Comment:
Could someone from the Meteor Team (@adi-de-masi, @macrozone, ...) write some sentences here?